lib/client/components/Location.jsx

```diff
@@ -1,5 +1,6 @@
 'use strict';
 import React from 'react';
+var Geolocation = require('../util/Geolocation');
```

Comment:
Does Jest not support mocking a browser global ? Seems like a pain to have to wrap everything, but if this is the way it is done (i.e. this is the recommended way in their docs or something) then i guess i could give it a chance